pymc/tests/distributions/test_timeseries.py

```diff
@@ -853,9 +856,9 @@ def sde_fn(x, k, d, s):
                 "y", dt=0.02, sde_fn=sde_fn, sde_pars=sde_pars, init_dist=init_dist, **kwargs
             )
 
-        y_eval = draw(y, draws=2)
+        y_eval = draw(y, draws=2, random_seed=RANDOM_SEED)
         assert y_eval[0].shape == (batch_size, steps)
-        assert not np.any(np.isclose(y_eval[0], y_eval[1]))
+        assert np.any(~np.isclose(y_eval[0], y_eval[1]))
```

Comment:
```python 
np.any(~np.isclose([1, 2, 3, 4, 5], [1, 2, 3, 4, 5]))   # -> False since all the same
np.any(~np.isclose([5, 4, 3, 2, 1], [1, 2, 3, 4, 5]))   # -> True since at least one different 
np.any(~np.isclose([1, 2, 3, 4, 5], [6, 7, 8, 9, 10]))  # -> True since all different

Reply:
Do you reckon this check is better, than checking all are different?

Reply:
I actually had it fail on my end where checking that all are different (previous test). That is, the two arrays were the same at at least 1 spot. In the case of an intersection between the two ys, I saw this better capturing that they are different

Reply:
Yes. There are two changes: 
1. Deterministic because of the random seed
2. Change in the assert which, I thought, better checks that the two draws are different based on the logic described. If you disagree, I will change back

Reply:
It still passes under the seed and the previous assert though but so can any test catered to do so

Reply:
I don't have a strong preference, but all values differently is certainly stronger evidence that it's being random :D

Reply:
Fair. 😃